extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -580,7 +604,24 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
               visiblePercentageMin: 50,
               continuousTimeMin: 1,
             });
-    return superReturnValue;
+    return frameLoadPromise.then(result => {
+      if (this.isFluid_) {
+        // If this is a Fluid slot, we must send an initial message to
+        // safeframe.
+        this.connectFluidMessagingChannel();
+      }
+      return Promise.resolve(result);
```

Comment:
I think you can just return result here... no need to wrap in resolve

Reply:
Done.